bqplot/pyplot.py

```diff
@@ -346,6 +346,70 @@ def axes(mark=None, options={}, **kwargs):
     return axes
 
 
+def grids(fig=None, value='solid'):
+    """Sets the value of the grid_lines for the axis to the passed value.
+    The default value is `solid`.
+    Parameters
+    ----------
+    fig: Figure or None(default: None)
+        The figure for which the axes should be edited. If the value is None,
+        the current figure is used.
+    value: {'none', 'solid', 'dashed'}
+        The display of the grid_lines
+    """
+
+    if fig is None:
+        fig = current_figure()
+    for a in fig.axes:
+        a.grid_lines = value
+
+
+def hline(level=0., fig=None, preserve_domain=True, **kwargs):
+    """Draws a horizontal line at the given level. By default, draws a line at
+    y=0.
+    Parameters
+    ----------
+    level: float (default: 0)
+        The level at which to draw the horizontal line
+    fig: Figure or None
+        The figure for which the axes should be edited. If the value is None,
+        the current figure is used.
+    preserve_domain: boolean (default: True)
+        If true, the line does not affect the domain of the 'y' scale.
+    """
+    default_colors = kwargs.pop('colors', ['dodgerblue'])
+    default_width = kwargs.pop('stroke_width', 1)
+    if fig is None:
+        fig = current_figure()
+    sc_x = fig.scale_x
+    plot([0., 1.], [level, level], scales={'x': sc_x}, preserve_domain={'x': True,
+         'y': preserve_domain}, axes=False, colors=default_colors,
+         stroke_width=default_width, update_context=False)
+
+
+def vline(level=0., fig=None, preserve_domain=True, **kwargs):
```

Comment:
Should the `level` be set to 0 by default? Or should it be midway through the `Figure` scale?